.eslintrc.yaml

```diff
@@ -223,7 +223,7 @@ rules:
   ########################
 
   import/export: error
-  import/extensions: [warn, never]
+  import/extensions: off
```

Comment:
This is heavily configurable, so I wouldn't 'off' it - https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md

If we want to only allow extensions for `mjs` files, then AFAICT you can use `[warn, never, { mjs: always }]`.

If, as you say elsewhere, we should use it everywhere but we don't yet, I'd move it to the unfixed rules list and do `[warn, always]` instead.

I guess you could technically even do both? So for now the fixed list could have the first, but the unfixed the second?

Reply:
Sure, I'll look into adding both and see how that works. :)

Reply:
I added `[error, always]` to the unfixed rule list (error because a missing extension will actually cause an error in Node).

I'm not sure if the other rule is helpful, because we really want to enforce all extensions *within* .mjs files, not just extensions *to* .mjs files.